P1/Project Brief/entertainment-web-app/server/src/routes/shows.router.js

```diff
@@ -0,0 +1,15 @@
+import { Router } from 'express';
+import { getShows , postShow } from "../controllers/show.controller.js" ;
+import { upload } from '../middlewares/multer.middleware.js';
+
+const router = Router();
+
+router.route('/shows').get( 
+    getShows
+);
+
+router.route('/post/shows').post( 
```

Comment:
not recommended to include the HTTP action being performed in the route path. this could just be "/show"

Reply:
Okay sir, will change